### DISCUSSION TITLE
Does Sequelize support Mysql 8 window functions?

### DISCUSSION THREAD

**Discussion by qmenchaca** [sequelize/sequelize#16196] [Help & Questions]
Just curious if Sequelize natively supports Mysql 8.0 window functions?

https://dev.mysql.com/doc/refman/8.0/en/window-functions-usage.html

I haven't found anything in docs or in StackOverflow, so thought I'd ask here. Thanks!

**Comment by ephys** [comment]
You'd need to use raw SQL for that ([v7](https://sequelize.org/docs/v7/querying/raw-queries/), [v6](https://sequelize.org/docs/v6/core-concepts/raw-queries/)). You can use fetch extra attributes with raw sql like this: https://sequelize.org/docs/v7/querying/select-in-depth/#extra-attributes
